core/src/main/kotlin/io/specmatic/core/value/JSONObjectValue.kt

```diff
@@ -86,6 +86,21 @@ data class JSONObjectValue(val jsonObject: Map<String, Value> = emptyMap()) : Va
 
     fun findFirstChildByName(name: String): Value? =
         jsonObject[name]
+
+    fun objectWithMandatoryKeys(
```

Comment:
Do we still need this method? I believe it's not being called from anywhere.

Reply:
Agree, i've dropped the method now